solid_storage/src/lib.rs

```diff
@@ -1,3 +1,45 @@
+extern crate zbox;
```

Comment:
**deprecated!**
`extern crate zbox;` is not needed anymore.
The use statement is enough `use zbox::{init_env, OpenOptions, RepoOpener};`.